tests/Unit/Form/Builder/Sorter/SizeSorterTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusMeilisearchPlugin\Tests\Unit\Form\Builder\Sorter;
+
+use PHPUnit\Framework\TestCase;
+use Setono\SyliusMeilisearchPlugin\Form\Builder\Sorter\SizeSorter;
+
+final class SizeSorterTest extends TestCase
```

Comment:
We need some realistic sizes here. Here's a list of realistic choices:

```
XS
S
M
L
XL
XXL
XXXL
40
41
42
43
44
45
46
48
50
52
54
56
XXS
4XL
34/36
38/32
34/30
38/34
33/36
S/M
36/36
27/30
27/32
32/36
31/36
L/XL
28/34
40 - 47
36/30
M/L
36/34
29/34
33/30
33/34
ONE SIZE
29/32
30/32
31/32
28/30
33/32
34/32
32/34
32/32
34/34
29/30
36/32
31/34
30/34
31/30
28/32
30/30
32/30
44,5
45.5
8
9
10
11
12
28
29
30
31
32
33
34
36
38
58
85
90
95
105
XS/S
```

Reply:
Notice the `44,5`, hehe, which is a Danish decimal point, so it should be `44.5`